src/styles/components/_venue.scss

```diff
@@ -111,14 +118,18 @@
 }
 
 .venue_slogan {
+    h2 {
+        font-size: 28px;
+    }
     width: 100%;
     text-align: center;
     display: inline-block;
     margin: 0 auto;
-    padding: 60px 60px 0px 60px;
+    padding: 60px 20px 60px 20px;
     background-color: $color-white;
 
     @media (min-width: 1023px) {
```

Comment:
I think here we can set the `min-width: 768px;` so it can apply the rules for Tablet as well and match Figma designs